Tests/MapboxSearchUITests/MockServerUITestCase.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 
-class MockServerUITestCase: BaseTestCase {
-    let server = MockWebServer()
+class MockServerUITestCase<Mock: MockResponse>: BaseTestCase {
```

Comment:
There are a lot of `MockServerUITestCase<SBSMockResponse>` declaration. It might make sense to declare `MockServerUITestCase<SBSMockResponse>` as a non-generic type.

Reply:
👍